src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+
+    @Override
+    public List<String> read(String pathToFile) {
+        try {
+            return Files.readAllLines(Path.of(pathToFile));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file: " + pathToFile);
```

Comment:
When catching `IOException`, consider including the original exception in the `RuntimeException` by using `throw new RuntimeException("Can't read from file: " + pathToFile, e);`. This will preserve the stack trace and provide more context for debugging.